caktus_primer/static/stylus/form.styl

```diff
@@ -34,7 +34,8 @@ button.cancel
 button.danger
     button-style("elem-danger")
 
-fieldset
+fieldset,
+fieldset div
```

Comment:
I'd really like to avoid the `div` and keep things directly under the `fieldset`. The least complicated markup we can prescribe the better.